### ISSUE TITLE
Problems when touch an element inside the CardView

### ISSUE THREAD

**Issue by jolugarbe** [AndreiMisiukevich/CardView#118]
Hi,

First of all thank you very much for this plugin.

I have a problem that I don't know how to resolve. When I touch inside the CardView on elements that I have added TapGestureRecognizer sometimes not work well. I some devices works perfect but others like Samsung Galaxy S8 works very bad and it is necessary touch sometimes to open o make the corresponding action.

I changed the CardView by ListView and the touch is perfect on all devices to check if is my code or not.

This only happens on some Android devices. On iOS works perfect always.
Any idea please?

My english is not very good, sorry.

**Comment by AndreiMisiukevich** [comment]
@jolugarbe hi) Thanks for feedback
Try please set MoveThresholdDistance = 20

**Comment by AndreiMisiukevich** [comment]
Is it possible to reproduce on android emulator? Or it's just device issue

**Comment by jolugarbe** [comment]
Thank you very much!! 

I have set MoveThresholdDistance = 20 and now works perfect on Galaxy S8 so surely works well on other devices that happened the same. On android emulator can't reproduce it.

Thanks again!

**Comment by AndreiMisiukevich** [comment]
@jolugarbe could you please set less value? (default value is 3), try please with 5,6,7 etc.
It would be great to know, what value is most acceptable =)

**Comment by AndreiMisiukevich** [comment]
@jolugarbe if you like this plugin, you may set *star* )

**Comment by AndreiMisiukevich** [comment]
@jolugarbe have you investigated it ? )

**Comment by jolugarbe** [comment]
Sorry, this monday I will check it like you say from 3 until 20 and I will comment you here.

**Comment by jolugarbe** [comment]
Hi,
I have checked it with Samsung S8 that it is the device with I had these problemas and my feedback is next:
- Setting between 3 and 6 there are times that the touch does not detect well and I must press again.
- Setting between 7 and 9 is almost perfect. Maybe I have the problem 1 time between 15-20 touchs.
- Setting 10 onwards no problem.

I repeat that is only my feedback with this device. I don't know what happens with others.
Thank you very much for your help. Solved my problem!
I have set star this plugin :)
